ga4gh/backend.py

```diff
@@ -10,11 +10,13 @@
 import json
 import random
 
+
```

Comment:
Is this extra blank line intentional? We don't have this in any of the other files, so I would remove it in the interest of consistency (unless there's a good argument for it).